packages/openapi-generator/src/parser/document.spec.ts

```diff
@@ -54,7 +54,7 @@ describe('parseOpenApiDocument()', () => {
         packageName: '@sap/cloud-sdk-openapi-test-service',
         directoryName: 'test-service'
       },
-      { strictNaming: false, schemaPrefix: '' }
+      { strictNaming: false, schemaPrefix: '', resolveExternal: true }
```

Comment:
Maybe as an improvement we can make as a follow-up, define defaultOptions for the function

```
defaultOptions = { strictName: true, schemaPrefix: '', resolveExternal: true }

export async function parseOpenApiDocument(
  fileContent: OpenAPIV3.Document,
  serviceOptions: ServiceOptions,
  options: ParserOptions = defaultOptions
): Promise<OpenApiDocument> {
```